gulpfile.js

```diff
@@ -173,7 +151,7 @@ gulp.task('release', function(done) {
      */
     const releaseWin = function(arch, os) {
         // TRICKY: the iss script cannot take the .exe extension on the file name
-        var file = `tS_${p.version}-${p.build}_win_x${arch}`;
+        var file = `BTT-${p.version}-${p.build}-win-x${arch}`;
```

Comment:
I messed this up. this line should read:

    var file = `BTT-Writer-${p.version}-${p.build}-win-x${arch}`;

Reply:
done